.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>trunk</name>
```

Comment:
This is a internal settings file used by Eclipse. I could not find the preferences to change its behavior.